<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>5</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>ДУСы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,8">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1,1,2">
            <item>
             <widget class="QTableWidget" name="tableCntMt">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QHeaderView::section {
    background-color: rgb(215, 215, 215); 
    border: 1px solid #000; 
}
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="autoScroll">
               <bool>true</bool>
              </property>
              <property name="autoScrollMargin">
               <number>25</number>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::SolidLine</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="cornerButtonEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>15</number>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>35</number>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>15</number>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>19</number>
              </attribute>
              <attribute name="verticalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <row>
               <property name="text">
                <string>G</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>AF</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>BF</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>AT</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>BT</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>AD</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>BD</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>CD</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>AK</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>BK</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>CK</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>AV</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>BV</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>CV</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>DV</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>C1</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>C2</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <column>
               <property name="text">
                <string>X</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>false</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Y</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>false</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Z</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>false</bold>
                </font>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string/>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="textAlignment">
                <set>AlignTrailing|AlignVCenter</set>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string/>
               </property>
               <property name="font">
                <font>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="flags">
                <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsAutoTristate</set>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Применить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableCoefs">
              <property name="styleSheet">
               <string notr="true">QHeaderView::section {
    background-color: rgb(200, 200, 190); 
    border: 1px solid #000; 
}
</string>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <row>
               <property name="text">
                <string>tau</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>D</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>K</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>V</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <row>
               <property name="text">
                <string>Km</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </row>
              <column>
               <property name="text">
                <string>X</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Y</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Z</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="plainText">
             <string>ыаффыааыфафы</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget" native="true"/>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true"/>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>Акселерометры</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Info about mt+acc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>20</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="aboutProgramMenu">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="aboutProgramMenu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Загрузить комплект</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сохранить комплект</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>выход2</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>выход 3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
